src/components/Portfolio/LoanList.vue

```diff
@@ -0,0 +1,234 @@
+								<div class="tw-text-secondary">
+									{{ getPrintableStatus(loan.status) }}
+								</div>
+							</td>
+							<td class="tw-text-right tw-px-2">
+								<div>
+									<div class="tw-mb-1">
+										${{ loan.userProperties.loanBalance.amountPurchasedByLender }}
+									</div>
+									<div class="tw-mb-1 tw-text-secondary">
+										{{ formatDate(loan.terms.disbursalDate) || '(Endpoint TBD)' }}
+									</div>
+								</div>
+							</td>
+							<td class="tw-text-right tw-px-2">
+								<div>
+									<div>
+										${{ loan.terms.loanAmount }}
+									</div>
+								</div>
+							</td>
+							<td class="tw-text-right tw-px-2">
+								<div>
+									{{ loan.lenderRepaymentTerm || '-' }} mos
+								</div>
+							</td>
+							<td class="tw-text-right tw-px-2">
+								<div>
+									<div class="tw-mb-1">
+										${{ loan.userProperties.loanBalance.amountRepaidToLender }} repaid to you
+									</div>
+								</div>
+							</td>
+							<td class="tw-max-w-[150px] tw-whitespace-normal tw-break-words tw-px-2">
+								<div class="tw-items-center">
+									<template v-if="loan.teams?.values?.[0]">
+										<img
+											v-if="loan.teams.values[0].image?.url"
+											:src="loan.teams.values[0].image.url"
+											:alt="`${loan.teams.values[0].name} team image`"
+											class="tw-w-5 tw-h-5"
+										>
+										<span>{{ loan.teams.values[0].name }}</span>
+									</template>
+									<span v-else>-</span>
+								</div>
+							</td>
+						</tr>
+					</tbody>
+				</table>
+			</div>
+			<div class="scroll-gradient"></div>
+		</div>
+	</div>
+</template>
+
+<script>
+import { KvFlag, KvLoadingPlaceholder } from '@kiva/kv-components';
+
+export default {
+	name: 'LoanList',
+	inject: ['cookieStore'],
+	props: {
+		loans: {
+			type: Array,
+			default: () => []
+		},
+		loading: {
+			type: Boolean,
+			default: true
+		}
+	},
+	components: {
+		KvFlag,
+		KvLoadingPlaceholder
+	},
+	methods: {
+		formatDate(date) {
+			if (!date) return '';
+			return new Date(date).toLocaleDateString('en-US', {
+				month: 'short',
+				day: 'numeric',
+				year: 'numeric'
+			});
+		},
+		getPrintableStatus(rawStatus) {
+			const mapping = {
+				fundraising: 'Fundraising',
+				funded: 'Funded',
+				raised: 'Raised',
+				payingBack: 'Paying Back',
+				ended: 'Ended',
+				delinquent: 'Delinquent',
+				defaulted: 'Defaulted',
+				refunded: 'Refunded',
+				expired: 'Expired'
+			};
+			return mapping[rawStatus] || rawStatus;
+		},
+	},
+	computed: {
+		getPartnerUrl() {
+			return partnerId => `/about/where-kiva-works/partners/${partnerId}`;
+		}
+	},
+	data() {
+		return {
+			placeholders: [
+				{ span: 4 },
+				{ span: 1 },
+				{ span: 1, marginLeft: true },
+				{ span: 1, marginLeft: true },
+				{ span: 1, marginLeft: true },
+				{ span: 2, marginLeft: true },
+				{ span: 2 }
+			]
+		};
+	}
+};
+</script>
+
+<style scoped>
+.scroll-gradient {
```

Comment:
Another pending decision that needs to be run by product/design is how we want to handle the issue of this table being wider than the container containing it for the latest table component design.